src/library_webgpu.js

```diff
@@ -2052,13 +2052,14 @@ var LibraryWebGPU = {
     }
 
     var data = _memalign(16, mapped.byteLength);
-    HEAPU8.fill(0, data, mapped.byteLength);
     if (bufferWrapper.mapMode === {{{ gpu.MapMode.Write }}}) {
+      HEAPU8.fill(0, data, mapped.byteLength);
```

Comment:
Can you replace this with a call to `zeroMemory` to be more explicit?    (you will also need to add `$zeroMemory` to the `__deps` of this function).

Reply:
I'm not very familiar with this code, what does HEAPU8.fill do versus zeroMemory?
Note: the HEAPU8.fill was already there in the Write case, this is essentially unchanged.

Reply:
No need to include that change in this PR if you don't want to..  I can make a separate PR to use zeroMemory here.

Reply:
A separate PR would be better, thanks :)

I'm not comfortable enough with the codebase to add dependencies myself